server/stream.go

```diff
@@ -4134,19 +4129,21 @@ func (mset *stream) signalConsumers(subj string, seq uint64) {
 	mset.clsMu.RLock()
 	defer mset.clsMu.RUnlock()
 
-	for _, o := range mset.cList {
-		o.mu.Lock()
-		if o.isLeader() && o.isFilteredMatch(subj) {
-			if seq > o.npcm {
-				o.npc++
-			}
-			if o.mset != nil {
-				if o.isPushMode() && o.active || o.isPullMode() && !o.waiting.isEmpty() {
-					o.signalNewMessages()
-				}
-			}
-		}
-		o.mu.Unlock()
+	if mset.csl == nil {
+		return
+	}
+
+	r := mset.csl.Match(subj)
+	if len(r.psubs) == 0 {
+		return
+	}
+	// Encode the sequence here.
+	var eseq [8]byte
+	var le = binary.LittleEndian
+	le.PutUint64(eseq[0:], seq)
```

Comment:
That's the same? 
Let's use the same syntax for consistency, as it can confuse someone :). 
```suggestion
	le.PutUint64(eseq[:], seq)
```

Reply:
Will fix.